services/motion/builtin/builtin.go

```diff
@@ -314,14 +353,18 @@ func (ms *builtIn) MoveOnGlobe(
 	// and exits when something is read from the success channel
 	for {
 		if err := ctx.Err(); err != nil {
+			cancelFn()
 			return false, err
 		}
 		select {
 		case <-ctx.Done():
+			cancelFn()
```

Comment:
Not sure why these cancels are being added here.  Can't we just defer once?

Reply:
Given this is going to change soon, I'll just remove these